cloud/pkg/edgecontroller/config/config.go

```diff
@@ -250,10 +249,6 @@ func InitConfigure() {
 			kc = ""
 			klog.Infof("can not get key controller.kube.kubeconfig, use default value %v", kc)
 		}
-		if km == "" && kc == "" {
-			errs = append(errs, fmt.Errorf("controller.kube.kubeconfig and controller.kube.master are not both setd"))
-		}
-
```

Comment:
where is the new logic to check one of the [kubeconfig, kube.master] must be set?

Reply:
if run cloudcore in container, both km and kc can be empty.